forui/lib/src/theme/style/zinc_style.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+import 'package:forui/forui.dart';
+
+/// Zinc style of [FThemeData].
+///
+/// The zinc style is based on [shadcn/ui](https://ui.shadcn.com/themes).
+extension FZincStyle on Never {
+  // TODO: Manually verify fields. Currently generated with a LLM.
+
+  /// The light theme of [FZincStyle].
+  static FThemeData light = FThemeData(
```

Comment:
Can this field be a constant?

Reply:
It can't be a constant, `BorderRadius` can't be a constant.